components/dashboard/src/service/public-api.ts

```diff
@@ -183,7 +190,7 @@ function createServiceClient<T extends ServiceType>(
                 const resolvedFlags = await Promise.all(
                     featureFlags.map((ff) =>
                         experimentsClient.getValueAsync(ff, false, {
-                            user,
+                            user: { id: userId || "" },
```

Comment:
that's rather unfortunate, we used email to enable only for Gitpod team on Cloud, could we keep email at least somehow?

Reply:
'fixed